ComponentKit/Core/CKComponentViewAttribute.h

```diff
@@ -194,11 +187,8 @@ namespace CK {
 
       void performSetter(id object, SEL setter) const
       {
-        const CachedSetter &set = CachedSetterInvocation(object, setter);
-        T value = _value;
-        [set.invocation setArgument:&value atIndex:2];
-        [set.invocation setSelector:setter];
-        [set.invocation invokeWithTarget:object];
+        const auto setterIMP = (void (*)(id, SEL, T))[object methodForSelector:setter];
```

Comment:
Added a test doing so and it works!

Is the `IMP` ignoring the typecast type (some kind of type erasure) and still taking in `_value` by reference?